src/docs/metrics/contract-interacting-address-count/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Interacting Address Count
+author: Yury
+date: 2022-04-21
+description: Get interacting address count for a chain address / contract
+---
+
+## Definition
+
+Get interacting address count for a chain address / contract
+
+[Example of usage **interacting address count** on Sanbase](https://app.santiment.net/s/rk-8It-D)
+
+---
+
+## Access
+
+[Free Access](/metrics/details/access#free-access)
+
+---
+
+## API
+
+Available under the `interacting address count` name:
+
+```graphql
+query getMetric($metric: String!, $from: DateTime!, $to: DateTime!, $interval: interval, $transform: TimeseriesMetricTransformInputObject, $aggregation: Aggregation, $address: String, $includeIncompleteData: Boolean = true) {
+  getMetric(metric: $metric) {
+    timeseriesData(selector: {contractAddress: $address}, from: $from, to: $to, interval: $interval, transform: $transform, aggregation: $aggregation, includeIncompleteData: $includeIncompleteData) {
+      d: datetime
+      v: value
+    }
+  }
+}
+```
+
+Query variables:
+```json
+{
+  "address": "0x857086e5e3dc7bbc98edb1639b4ffd96a667d75a",
+  "from": "2021-10-21T18:59:59.999Z",
+  "interval": "4h",
+  "metric": "contract_interacting_addresses_count",
+  "to": "2022-04-21T18:59:59.999Z"
+}
+```
```

Comment:
Please follow the structure in the other metric examples. Providing query and arguments separately makes the query much less readable.